packages/svelte/src/internal/client/proxy.js

```diff
@@ -327,8 +327,18 @@ function update_version(signal, d = 1) {
  * @param {any} value
  */
 export function get_proxied_value(value) {
-	if (value !== null && typeof value === 'object' && STATE_SYMBOL in value) {
-		return value[STATE_SYMBOL];
+	try {
+		if (value !== null && typeof value === 'object' && STATE_SYMBOL in value) {
+			return value[STATE_SYMBOL];
+		}
+	} catch (err) {
```

Comment:
since we're not using `err` we can remove it

```suggestion
	} catch {
```